src/frameserver/PointGreyCam.cpp

```diff
@@ -539,7 +541,7 @@ void PointGreyCam<T>::setupAsyncTrigger(int trigger_mode,
         trigger_mode != 1  ||
         trigger_mode != 13 ||
         trigger_mode != 14) {
-        throw (rte("Trigger mode is unsupported."));
+      //throw (rte("Trigger mode is unsupported."));
```

Comment:
I think the issue is the the free-running check block

```
if (trigger_mode < 0 ) {
          use_trigger_ = false;
         return;
 }
```
must come __before__ all the trigger mode checks. In that case, the function will just return and not check for a valid trigger mode, and therefore, not throw when it finds its invalid (since I've designated mode -1 to be free running).